### PR TITLE
fix: Unclosed ~/ links

### PR DESCRIPTION
**Pull request by nschonni**
The issue tracker isn't open so I'll fix and report an issue at the same time.

The links starting with `~/` are probably better to replace according to the docs guide https://docs.microsoft.com/en-us/contribute/how-to-write-links#links-from-one-article-to-another

### CODE REVIEW

**Comment by PRMerger17** [comment]
@nschonni : Thanks for your contribution! The author, @douglaslMS, has been notified to review your proposed change.

**Comment by douglaslMS** [comment]
#sign-off

File: docs/2014/relational-databases/extended-events/monitor-system-activity-using-extended-events.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ manager: craigg
  [CREATE EVENT SESSION &#40;Transact-SQL&#41;](/sql/t-sql/statements/create-event-session-transact-sql)   
  [ALTER EVENT SESSION &#40;Transact-SQL&#41;](/sql/t-sql/statements/alter-event-session-transact-sql)   
  [DROP EVENT SESSION &#40;Transact-SQL&#41;](/sql/t-sql/statements/drop-event-session-transact-sql)   
- [Extended Events Catalog Views &#40;Transact-SQL&#41;](~/relational-databases/system-catalog-views/extended-events-catalog-views-transact-sql   
+ [Extended Events Catalog Views &#40;Transact-SQL&#41;](~/relational-databases/system-catalog-views/extended-events-catalog-views-transact-sql)
```
Comment (nschonni) [inline docs/2014/relational-databases/extended-events/monitor-system-activity-using-extended-events.md]: Ah, looks like this is missing the extension

**Comment by douglaslMS** [comment]
#sign-off

**Comment by MightyPen** [comment]
WRONG:
[Extended Events Catalog Views &#40;Transact-SQL&#41;](~/relational-databases/system-catalog-views/extended-events-catalog-views-transact-sql)
.
.
RIGHT - I THINK:
[Extended Events Catalog Views &#40;Transact-SQL&#41;](../system-stored-procedures/system-catalog-views/extended-events-catalog-views-transact-sql.md)

**Comment by douglaslMS** [comment]
@MightyPen That would work if these weren't in the 2014 folder, where there is no system-stored-procedures folder.

These links caused a build break and have been revised in another PR. Please let's not touch them any more for the moment!

**Comment by nschonni** [comment]
Yup, I flagged the issue, but it got merged before I fixed it

**Comment by MightyPen** [comment]
Oh, I did not notice the /2014/ node   :-(

**Pull request merged**